cli/wallet/multisig.go

```diff
@@ -26,16 +28,19 @@ func signStoredTransaction(ctx *cli.Context) error {
 	if !addrFlag.IsSet {
 		return cli.NewExitError("address was not provided", 1)
 	}
-	acc, err := getDecryptedAccount(ctx, wall, addrFlag.Uint160())
-	if err != nil {
-		return cli.NewExitError(err, 1)
-	}
 
 	tx, ok := c.Verifiable.(*transaction.Transaction)
 	if !ok {
 		return cli.NewExitError("verifiable item is not a transaction", 1)
 	}
 
+	dumpTransaction(ctx, tx)
+
```

Comment:
Ask for confirmation here? Or make it a separate command (unified with test-execution)?